.github/workflows/build.yml

```diff
@@ -11,6 +11,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+
+      - name: Cache node_modules
+        uses: actions/cache@v2
+        id: module-cache
+        with:
+          path: ./js/node_modules
+          key: ${{ runner.os }}-node-modules-${{ github.sha }}-${{ hashFiles('**/package-lock.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-node-modules-${{ github.sha }}-
```

Comment:
Shouldn't this also be `${{ runner.os }}-node-modules-${{ github.sha }}-${{ hashFiles('**/package-lock.lock') }}`?

Reply:
This is a set of keys to match as a fallback if the normal key doesn't exist.

This lets us also restore the previous node modules if we add new deps/update deps, so we only install deps that changed, just like locally.

Took me a few hours of research to understand this... It's not very clear.